assets/app/views/catalog/images.html

```diff
@@ -1,13 +1,13 @@
 <div ng-controller="ProjectController">
   <div ng-controller="CatalogImagesController">
     <div class="container">
+      <ol class="breadcrumb">
+        <li><a href="{{project | projectOverviewURL}}">{{project | displayName}}</a></li>
```

Comment:
`{{projectName | projectOverviewURL}}`, otherwise the link is invalid until the project loads